src/wifi/AdafruitIO_MKR1010.cpp

```diff
@@ -0,0 +1,77 @@
+/*!
```

Comment:
Could you please combine `AdafruitIO_MKR1010.cpp` and `AdafruitIO_MKR1010.h` into one file - `AdafruitIO_MKR1010.h`?

We do this with the AdafruitIO_AIRLIFT class, which is similar: https://github.com/adafruit/Adafruit_IO_Arduino/blob/master/src/wifi/AdafruitIO_AIRLIFT.h